$MirOS: ports/emulators/zsnes/patches/patch-src_ui_asm,v 1.1 2005/10/02 19:25:35 tg Exp $
--- src/ui.asm.orig	Fri Jan 14 13:10:20 2005
+++ src/ui.asm	Mon Nov  3 17:11:27 2008
@@ -289,7 +289,7 @@ NEWSYM setnoise
     ret
 
 SECTION .data
-.samplenoise
+.samplenoise:
   db 27,232,234,138,187,246,176,81,25,241,1,127,154,190,195,103,231,165,220,238
   db 232,189,57,201,123,75,63,143,145,159,13,236,191,142,56,164,222,80,88,13
   db 148,118,162,212,157,146,176,0,241,88,244,238,51,235,149,50,77,212,186,241
@@ -1237,7 +1237,7 @@ NEWSYM gotoroot, db '\',0
 
 SECTION .bss
 NEWSYM InitDir, resb 512
-NEWSYM LoadDir, resb 512
+NEWSYM LoadDir, resb 1024
 
 SECTION .text
 
